rfcs/0148-pipe-operator.md

```diff
@@ -0,0 +1,327 @@
+---
+feature: pipe-operator
+start-date: 2023-05-23
+author: @piegamesde
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
```

Comment:
The more shepherds the merrier!

```suggestion
shepherd-team: @roberth @iFreilicht @illustris @adrian-gierakowski 
shepherd-leader: @iFreilicht 
```

@iFreilicht I hope you don't mind being leader, just means to make sure the RFC keeps moving forward, e.g. by scheduling meetings

Reply:
Yes I'm cool with that :)

Reply:
RFC Steering Committee accepts proposed nominations!

Reply:
@piegamesde Can you merge the proposed change? Then we could maybe have schedule a kick-off and all get up to speed on what has to happen next.

Reply:
Btw there might be some confusion: I already proposed this change on behalf of the steering committee originally. In today's meeting we just wanted to reconfirm that these are the shepherds.